src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -130,21 +131,10 @@ private String convertKey(Schema keySchema, Object key) {
           break;
         case FAIL:
         default:
-          throw new DataException(
```

Comment:
not necessary change for this PR, if we want to do a refactor, let's go in a separate PR and target `11.0.x`